ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -340,4 +353,38 @@ public void shouldCloseInternallyCreatedTopicClientOnClose() {
     // Then:
     verify(topicClient);
   }
+
+
+  @Test
+  public void shouldParseMultipleStatements() {
+    final String statementsString = readStatementsFromFile(
+        "src/test/resources/SampleMultilineStatements.sql");
```

Comment:
have you committed this file? i don't see it

Reply:
Forgot to add it. Just added it.